Backend/NoCode.FlowerShop.Api/Controllers/FlowerArrangementsController.cs

```diff
@@ -43,4 +46,16 @@ public async Task<IActionResult> GetFlowerArrangementById(Guid id)
             result => Ok(_mapper.Map<GetFlowerArrangementByIdResponse>(result)),
             errors => Problem(errors));
     }
+
+    [HttpDelete("{id:guid}")]
+    [AuthorizeRoles(UserRole.Administrator)]
+    public async Task<IActionResult> DeleteFlowerArrangement(Guid id)
+    {
+        var command = new DeleteFlowerArrangementCommand(id);
+        var result = await _mediator.Send(command);
+
+        return result.Match(
+            result => NoContent(),
```

Comment:
There is misalignment between DELETE Flower endpoint, that returns HTTP 200 when deleted. I don't think this is a big issue (especially in the context of current deadlines :D). But it breaks consistency a little.

Reply:
I believe that DELETE endpoint should return HTTP 204 response code (NoContent) because the response does not return any entity. 
It is a bit strict way of following HTTP protocol:
https://www.rfc-editor.org/rfc/rfc9110.html#section-9.3

Reply:
Oh didn't know there is a convention for that. Thanks for sharing. Anyway it means we have some endpoints to refactor now :D But I guess we will make it a tech debt due to deadlines unless you think otherwise

Reply:
I think you should raise an issue so it would not get lost. If we find the time we will fix it - it is quite a minor detail overall.